docs/reference/context/environment.schema.json

```diff
@@ -137,6 +137,16 @@
                     "title": "Internet service provider",
                     "type": "string",
                     "description": "The name of the user's internet service provider."
+                },
+                "xdm:locale": {
```

Comment:
You don't need this since XDM is valid JSON-LD, you should be using the standard `@language` value.